### ISSUE TITLE
NoSemicolons error message wrong

### ISSUE THREAD

**Issue by NeQuissimus** [scalacenter/scalafix#660]
Write some code with semicolons and check with `noSemicolons`.

```scala
src/main/scala/PortHelper.scala:11:38: error: [DisableSyntax.noSemicolons] noSemicolons is disabled
    Try { val s = new ServerSocket(0); val port = s.getLocalPort; s.close; port }.getOrElse(findOpenPort)
```

What it means to tell us is that `noSemicolons` is *enabled* (or semicolons are disabled)

**Comment by olafurpg** [comment]
Thanks for reporting! I agree the message is confusing. Would it make more sense to report "semicolons are disabled"?

**Comment by NeQuissimus** [comment]
I think either way is fine :) I figure all the "DisableSyntax" modules probably say "X is disabled", which ever way is easier :)

**Comment by olafurpg** [comment]
Fixed in #738 by @tanishiking  🎉
